server/core/region.go

```diff
@@ -567,7 +567,8 @@ func GenerateRegionGuideFunc(enableLog bool) RegionGuideFunc {
 			// Once flow has changed, will update the cache.
 			// Because keys and bytes are strongly related, only bytes are judged.
 			if region.GetRoundBytesWritten() != origin.GetRoundBytesWritten() ||
-				region.GetRoundBytesRead() != origin.GetRoundBytesRead() {
+				region.GetRoundBytesRead() != origin.GetRoundBytesRead() ||
+				region.flowRoundDivisor != origin.flowRoundDivisor {
```

Comment:
How about `region.flowRoundDivisor < origin.flowRoundDivisor`?  To reduce unnecessary update.